src/sig_stateful/lms/external/Makefile

```diff
@@ -0,0 +1,121 @@
+AR = /usr/bin/ar
```

Comment:
Hardcoded compiler/assembler paths. Doesn't look right.

Reply:
Agreed but, this makefile will not be used in liboqs. This is as-is, untouched from the LMS repo. 
I guess the preferece is to only commit files that will be used? I can also remove the test files and migrate them later to oqs test directories.

Reply:
> I guess the preferece is to only commit files that will be used? 

Most definitely -- there shouldn't be anything in OQS that is useless.